### PR TITLE
[Wallet] keep awake verification loading screen + fix upgrade screen

### LINKED ISSUE #5001: [Wallet] Keep phone away from sleeping during verification countdown
### Expected Behavior

Screen is always active when user sees a countdown.

### Current Behavior

Phone will go to sleep respecting user setting while user waits for attestation messages been created

### PR DESCRIPTION
**Pull request by i1skn**
### Description

Keep awake verification loading screen.

### Other changes

Fix design in `Upgrade` screen, fix it for iOS.

### Tested

iOS and Android physical devices

### Related issues

- Fixes https://github.com/celo-org/celo-monorepo/issues/5001

### Backwards compatibility

Yes

### Screenshot 
Simulator Screen Shot - iPhone 11 Pro - 2020-09-22 at 18 43 43

### CODE REVIEW

**Comment by aaronmgdr** [comment]
That language is pretty scary.  Is that what we want? @nityas

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
Based on the Valora voice guidelines Im going to say we should not merge this until we have wording that fits those guidelines. 

possible suggestion
"Valora is now more secure." 

even that might not be right.  


https://docs.google.com/document/d/1CbB2r2tmdLXKrzxMe5b7qm1SziSDkP8w7s7pfhSKWEU/edit#

**Review by jmrossy — COMMENTED** [review]
Code changes LGTM but like @aaronmgdr said would be good to get confirmation on wording while we're editing this screen anyway. Initial version was hastily put together.

**Comment by nityas** [comment]
Thanks for the flag
Wording rec:
"App Update Available" "Your current version is out of date, please update to the latest version" @i1skn @aaronmgdr @jmrossy

**Comment by i1skn** [comment]
@aaronmgdr @jmrossy I've updated the wording, PTAL

**Review by aaronmgdr — APPROVED** [review]

File: packages/mobile/src/app/UpgradeScreen.tsx [hunk 1]
```diff
@@ -2,21 +2,24 @@ import FullscreenCTA from '@celo/react-components/components/FullscreenCTA'
...
+import { headerWithCloseButton } from 'src/navigator/Headers.v2'
+import { navigateToWalletStorePage } from 'src/utils/linking'
 
 type Props = WithTranslation
 
 class UpgradeScreen extends React.Component<Props> {
-  static navigationOptions = { header: null }
+  static navigationOptions = {
+    ...headerWithCloseButton,
+  }
```
Comment (jeanregisser) [inline packages/mobile/src/app/UpgradeScreen.tsx:13]: Looks like we didn't want this screen to be closed.
Sure we want to change this now?

File: packages/mobile/src/utils/linking.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 const TAG = 'utils/linking'
 
-export function navigateToWalletPlayStorePage() {
-  navigateToURI(`market://details?id=${DeviceInfo.getBundleId()}`)
+export function navigateToWalletStorePage() {
+  if (Platform.OS === 'android') {
+    navigateToURI(`market://details?id=${DeviceInfo.getBundleId()}`)
+  } else {
+    navigateToURI(`https://apps.apple.com/app/id${APP_STORE_ID}`)
+  }
```
Comment (jeanregisser) [inline packages/mobile/src/utils/linking.ts:13]: 👍

**Review by jeanregisser — COMMENTED** [review]
Great! 👍 
See comment below.

File: packages/mobile/src/app/UpgradeScreen.tsx (hunk 1 above)
Comment (i1skn) [reply packages/mobile/src/app/UpgradeScreen.tsx:13]: Well, otherwise there is no way to close this screen if you are not in the mood of updating atm

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine I have verified this Task on latest test flight build (25) & Android internal build Android internal build 1.2.0(1004294313) and observe the following: 
**Keep awake verification loading screen**
- Verify mobile screen awake when user is on confirming countdown screen while verifying phone number. 

**Fix design in Upgrade screen, fix it for iOS**
- Verify user is able to see "App update " page when user install old version 
- i am not able to see "App update" screen  with following steps: 

1. Installed 1.0.1 build from test flight > Sign in with existing account. 
2. Installed 1.0.1 build from play store > Sign in with existing account. 
3. Installed play store 1.0.1 build> walk through screens. 

 
**Note:** Currently, we have 1.2.0 internal build for Android and IOS. 

please let me know if i need to verify with another steps to get the screen

**Comment by tarikbellamine** [comment]
@jeanregisser how can Ankit prompt this?

**Pull request merged**